src/RepoAuditor/Plugins/GitHub/ClassicBranchProtectionQuery.py

```diff
@@ -90,7 +90,7 @@ class ClassicBranchProtectionQuery(Query):
 
     # ----------------------------------------------------------------------
     def __init__(self) -> None:
-        super(ClassicBranchProtectionQuery, self).__init__(
+        super().__init__(

```

Comment:
Can you verify when this change was introduced into Python? It's a nice change as long as it was introduced in or before Python 3.10.